src/openApi/v3/parser/getOperation.ts

```diff
@@ -125,6 +129,36 @@ export const getOperation = (
         const operationResults = getOperationResults(operationResponses);
         operation.errors = getOperationErrors(operationResponses);
         operation.responseHeader = getOperationResponseHeader(operationResults);
+        operation.responseType = getOperationResponseType(operationResults);
+
+        // Add 'Accept' header, if not set
+        const explicitAcceptHeader = parameters ? getOperationExplicitAcceptHeader(parameters.parametersHeader) : null;
+        if (!explicitAcceptHeader) {
+            const acceptHeader = getOperationImplicitAcceptHeader(operationResults);
+            if (acceptHeader) {
```

Comment:
I'm sure this will need some linting fixes but logic is easier to follow this way IMO.

```suggestion
        const acceptHeader = parameters ? getOperationExplicitAcceptHeader(parameters.parametersHeader) : getOperationImplicitAcceptHeader(operationResults);
        if (acceptHeader) {
```

Reply:
yup

Reply:
Reverted this.

Reason: https://github.com/lune-climate/openapi-typescript-codegen/pull/84/files#diff-77cc3bd44dd5161d8ceb51a1ad4c8b4e81a2c91ae914dd0681e6255bbed05015R137